File: twiml/voice/connect/virtualagent-6/virtualagent-6.6.x.py
from twilio.twiml.voice_response import Connect, VoiceResponse, VirtualAgent

response = VoiceResponse()
connect = Connect()
virtualagent = VirtualAgent(connector_name='uniqueName')
virtualagent.config(name='language', value='en-us')
virtualagent.config(name='voiceName', value='en-US-Wavenet-C')
connect.append(virtualagent)
response.append(connect)

print(response)

File: twiml/voice/connect/virtualagent-6/virtualagent-6.6.x.cs
using System;
using Twilio.TwiML;
using Twilio.TwiML.Voice;


class Example
{
    static void Main()
    {
        var response = new VoiceResponse();
        var connect = new Connect();
        var virtualagent = new VirtualAgent(connectorName: "uniqueName");
        virtualagent.Config(name: "language", value: "en-us");
        virtualagent.Config(name: "voiceName", value: "en-US-Wavenet-C");
        connect.Append(virtualagent);
        response.Append(connect);

        Console.WriteLine(response.ToString());
    }
}
